lib/fog/bin.rb

```diff
@@ -73,7 +73,6 @@ def collections
 require 'fog/bin/linode'
 require 'fog/bin/local'
 require 'fog/bin/bare_metal_cloud'
-require 'fog/bin/ninefold'
```

Comment:
You can find it [here](https://github.com/fog/fog/blob/master/lib/fog/bin/ninefold.rb). =)